tests/cupyx_tests/scipy_tests/signal_tests/test_signaltools.py

```diff
@@ -3,9 +3,15 @@
 import pytest
 
 import cupy
-import cupyx
 from cupy import testing
 
+import cupyx.scipy.signal  # NOQA
+
+try:
+    import scipy.signal  # NOQA
+except ImportError:
+    pass
```

Comment:
Removing these will cause *all* tests to fail because they won't be able to use `cupyx.scipy.signal` or `scipy.signal` (used as `scp.signal` later with `scp` either `cupyx.scipy` or `scipy`).

Reply:
I see. `@testing.numpy_cupy_allclose` imports just `scipy` and `cupyx.scipy` packages only, and `scipy.signal` and `cupyx.scipy.signal` can't be seen.